en/concept/mixing.md

```diff
@@ -250,18 +250,21 @@ Summing mixers are used for actuator and servo control.
 
 A summing (simple) mixer combines zero or more control inputs into a single actuator output.
 Inputs are scaled, and the mixing function sums the result before applying an output scaler.
+In the output scaler, also an actuator raise time limit can be specified (inverse of a slew rate).
 
 A simple mixer definition begins with:
 
 ```
 M: <control count>
-O: <-ve scale> <+ve scale> <offset> <lower limit> <upper limit>
+O: <-ve scale> <+ve scale> <offset> <lower limit> <upper limit> <raise time>
 ```
 
 If `<control count>` is zero, the sum is effectively zero and the mixer will output a fixed value that is `<offset>` constrained by `<lower limit>` and `<upper limit>`.
 
 The second line defines the output scaler with scaler parameters as discussed above.
-Whilst the calculations are performed as floating-point operations, the values stored in the definition file are scaled by a factor of 10000; i.e. an offset of -0.5 is encoded as -5000.
+Whilst the calculations are performed as floating-point operations, the values stored in the definition file are scaled by a factor of 10000; i.e. an offset of -0.5 is encoded as -5000. 
+The `<raise time>` can be used to limit the rate of change of an actuator.
```

Comment:
1. Is there a better name that `<raise time>`? I presume this also applies when going from max to min, so just as easily could be "drop time". Perhaps "traversal time"?
2. "Can be used to". Is it optional? Say what happens if it is omitted.
3. Is the rate limiting linear? If so, might be worth saying that - otherwise you might assume some other behaviour.